apps/scan/backend/scripts/copy_sheets.ts

```diff
@@ -69,11 +66,6 @@ function copySheet(store: Store, sheet: AcceptedSheet): void {
   ]);
 }
 
-function surfaceCastVoteRecordSyncModal(store: Store): void {
-  store.setIsContinuousExportOperationInProgress(true);
-  clearDoesUsbDriveRequireCastVoteRecordSyncCachedResult();
```

Comment:
I recently realized that this `clearDoesUsbDriveRequireCastVoteRecordSyncCachedResult` call doesn't do anything since the script and the actual running VxScan (if any) have their own instances of this cache (which is persisted in a local variable). As you'll see below, I now just print a prompt to restart VxScan if already running, to surface the prompt to sync cast vote records to the now out-of-sync USB drive.

As nice as it would be to have this script automatically surface the prompt on the running VxScan without requiring a restart, I'd need to make bigger changes to the `doesUsbDriveRequireCastVoteRecordSync` cache, which feels too risky for now given that I'm going to be cherry-picking this commit onto the release branch.